docs/content/preview/reference/drivers/java/postgres-jdbc-reference.md

```diff
@@ -221,35 +221,116 @@ Connection conn = DriverManager.getConnection(jdbc:postgresql://localhost:5433/y
 
-2. If you do not have access to the system `cacerts` Java truststore you can create your own truststore.
+1. If you do not have access to the system `cacerts` Java truststore you can create your own truststore.
 
     ```sh
     $ keytool -keystore ybtruststore -alias ybtruststore -import -file ca.crt
     ```
 
-3. Verify the `yugabytedb.crt` client certificate with `ybtruststore`.
+    Enter a password when you're prompted to enter one for your keystore.
+
+1. Export the truststore. In the following command, replace `<YOURSTOREPASS>` with the password you used for your keystore creation.
 
     ```sh
-    $ openssl verify -CAfile ca.crt -purpose sslclient tlstest.crt
+    $ keytool -exportcert -keystore ybtruststore -alias ybtruststore -storepass <YOURSTOREPASS> -file ybtruststore.crt
     ```
 
-4. Convert the client certificate to DER format.
+1. Convert and export to PEM format with `ybtruststore.pem`.
+
+   ```sh
+   $ openssl x509 -inform der -in ybtruststore.crt -out ybtruststore.pem
+   ```
+
+1. Verify the `yugabytedb.crt` client certificate with `ybtruststore`.
+
+    ```sh
+    $ openssl verify -CAfile ybtruststore.pem -purpose sslclient yugabytedb.crt
+    ```
+
+1. Convert the client certificate to DER format.
 
     ```sh
     $ openssl x509 –in yugabytedb.crt -out yugabytedb.crt.der -outform der
     ```
 
-5. Convert the client key to pk8 format.
+1. Convert the client key to pk8 format.
 
     ```sh
     $ openssl pkcs8 -topk8 -inform PEM -in yugabytedb.key -outform DER -nocrypt -out yugabytedb.key.pk8
     ```
 
-Create an `ssl` resource directory in your java application and copy over all the certificates. Update the connection string used by `DriverManager.getConnection` to include the ssl certificates.
+#### SSL certificates for a cluster in Kubernetes (Optional)
+
+Steps for configuring the JDBC client for server authentication in a Kubernetes cluster are as follows:
+
+1. Create a minikube cluster by adding `tls.enabled=true` to the command line described in [Quick start](../../../../quick-start/kubernetes/).
+
+   ```sh
+   $ kubectl create namespace yb-demo
+   $ helm install yb-demo yugabytedb/yugabyte \
+   --version {{<yb-version version="preview" format="short">}} \
+   --set resource.master.requests.cpu=0.5,resource.master.requests.memory=0.5Gi,\
+   resource.tserver.requests.cpu=0.5,resource.tserver.requests.memory=0.5Gi,\
+   replicas.master=1,replicas.tserver=1,tls.enabled=true --namespace yb-demo
+   ```
+
+1. Verify that SSL is enabled using `ysqlsh`.
+
+   ```sh
+    $ ysqlsh
+    ```
+
+    ```output
+    ysqlsh (11.2-YB-2.9.0.0-b0)
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    Type "help" for help.
+    ```
+
+1. Check for the key and certificate files in yb-tserver.
+
+   ```sh
```

Comment:
not sure sh is the right language

Reply:
checked for _kubectl_ commands in other files. We are using _sh_.